### PR TITLE
Add icons to Footer and change text

### LINKED ISSUE #334: Update the description in Footer
We need to update the description text in Footer, below the subscription section. The text should say at the end: **Icons by Font Awesome.**

[Link to Figma component](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=639%3A185&viewport=-454%2C233%2C0.514668345451355)

### LINKED ISSUE #339: Footer component on the storybook is not updated
**Description**
The footer on https://search.creativecommons.org/ has some social media icons but the [footer](https://cc-vocabulary.netlify.app/?path=/story/patterns-footer--default-story) component on the storybook does not.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #334 by @panchovm and fixes #339 by @makkoncept

## Description
Add missing icons for social networks and attribution. Also change text of icons authorship.

## Screenshots
footer-updated

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [x] My code follows the established code style of the repository.
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
@krysal can you please also update the snapshot for the Footer?

**Comment by krysal** [comment]
Sure, so are the changes on Footer ok?

**Comment by krysal** [comment]
@dhruvkb snapshot updated. I reduced the screenshot area to just the footer element, as suggested in #328.
Lastly, I believe there should be a break of line <br> between the email and the phone number to avoid what happens on wide screens, see screenshots:
footer
Captura de pantalla de 2020-04-27 09-30-35

**Review by dhruvkb — CHANGES_REQUESTED** [review]
> Lastly, I believe there should be a break of line between the email and the phone number to avoid what happens on wide screens

Yes. Good find @krysal! Please update this in the code and the screenshots.

**Comment by krysal** [comment]
@dhruvkb Done. To update the snapshot, I have to change the file permissions in my local environment; otherwise I get a `permission denied` error as in `vocabulary-ci / Snapshot tests`, not sure what is the right way to do it.

File: tests/snapshot/footer.test.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ describe('Footer', () => {
   it('Default', async () => {
     await goTo(page, 'patterns-footer--default-story')
     await pause(1000)
-    const component = await page.$('footer.main-footer')
-    const ss = await component.screenshot()
+    const component = await page.$('footer')
+    const image = await component.screenshot()
 
-    expect(ss).toMatchImageSnapshot({
+    expect(image).toMatchImageSnapshot({
```
Comment (dhruvkb) [inline tests/snapshot/footer.test.js]: These were unnecessary.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**